spec/voting_spec.rb

```diff
@@ -108,6 +108,30 @@
     end
   end
 
+  context "when topic is trashed" do
+    it "enqueues a job for releasing votes" do
```

Comment:
These tests are very fragile and test implementation vs actual behavior. 

Instead try `Jobs.run_immediately!` then trash a topic and check votes are gone, recover a topic check votes are back.

Reply:
@SamSaffron Yeah they were awful; I was able to find an issue with the code as well (actual behavior...)

Thanks Sam.